### PR TITLE
add info over where the bundle files are or will be written 

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**

- Small imp for just improve the logs. 

**Motivation for the change:**

- because it may not clear where the files have been  added

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [ ] Commit messages sensible and descriptive

Closes: https://github.com/operator-framework/operator-sdk/issues/2878

### CODE REVIEW

File: pkg/lib/bundle/generate.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ func GenerateFunc(directory, outputDir, packageName, channels, channelDefault st
 	} else if err != nil {
 		return err
 	} else {
-		log.Info("A bundle.Dockerfile already exists in current working directory")
+		log.Printf("A bundle.Dockerfile already exists in current working directory: %s", workingDir)
```
Comment (ecordell) [inline pkg/lib/bundle/generate.go:149]:
```suggestion
		log.Infof("A bundle.Dockerfile already exists in current working directory: %s", workingDir)
```
You can keep the log level with `Infof`

Comment (camilamacedo86) [reply pkg/lib/bundle/generate.go]: Done.

**Comment by camilamacedo86** [comment]
/test e2e-aws
--

**Comment by camilamacedo86** [comment]
Hi @ecordell, 

I need to rebase it with the master because it started to not pass in the CI. So, could you please give a hand here and add the lgtm again?

**Pull request merged**